features/measurements/impl/src/main/java/org/opennms/netmgt/measurements/impl/JEXLExpressionEngine.java

```diff
@@ -99,6 +99,9 @@ public void applyExpressions(final QueryRequest request, final FetchResults resu
         final Map<String, double[]> columns = results.getColumns();
         final int numRows = timestamps.length;
 
+        // Calculate the time span
+        jexlValues.put("__diff_time", numRows < 1 ? 0d : timestamps[numRows-1] - timestamps[0]);
```

Comment:
For example, ifInOctets is expressed in "bytes per second" (any counter is treated as a rate), if you want "bytes", you have to multiply by the amount of "seconds", not the amount of "milliseconds".